src/showingpreviously/cinemas/showcase.py

```diff
@@ -0,0 +1,169 @@
+        elif experience['ExternalId'] == 'FRE':
+            attributes['language'] = 'French'
+            attributes['subtitled'] = 'English'
+        elif experience['ExternalId'] == 'KORE':
+            attributes['language'] = 'Korean'
+            attributes['subtitled'] = 'English'
+        elif experience['ExternalId'] == 'POE':
+            attributes['language'] = 'Polish'
+            attributes['subtitled'] = 'English'
+        elif experience['ExternalId'] == 'CANTON':
+            attributes['language'] = 'Cantonese'
+            attributes['subtitled'] = 'Chinese and English'
+        elif experience['ExternalId'] == 'MANE':
+            attributes['language'] = 'Mandarin'
+            attributes['subtitled'] = 'English'
+        elif experience['ExternalId'] == 'TAMEN':
+            attributes['language'] = 'Tamil'
+            attributes['subtitled'] = 'English'
+        elif experience['ExternalId'] == 'MALAY':
+            attributes['language'] = 'Malayalam'
+            attributes['subtitled'] = 'English'
+        elif experience['ExternalId'] == 'HIE':
+            attributes['language'] = 'Hindi'
+            attributes['subtitled'] = 'English'
+        elif experience['ExternalId'] == 'FL-NPE':
+            attributes['language'] = 'Nepali'
+            attributes['subtitled'] = 'English'
+        elif experience['ExternalId'] == 'DANE':
+            attributes['language'] = 'Danish'
+            attributes['subtitled'] = 'English'
+        elif experience['ExternalId'] == 'RUSE':
+            attributes['language'] = 'Russian'
+            attributes['subtitled'] = 'English'
+        elif experience['ExternalId'] == 'ROME':
+            attributes['language'] = 'Romanian'
+            attributes['subtitled'] = 'English'
+        elif experience['ExternalId'] == 'LATV':
+            attributes['language'] = 'Latvian'
+            attributes['subtitled'] = 'English'
+        elif experience['ExternalId'] == 'ITALIAN':
+            attributes['language'] = 'Italian'
+            attributes['subtitled'] = 'English'
+        elif experience['ExternalId'] == 'HEBREW':
+            attributes['language'] = 'Hebrew'
+            attributes['subtitled'] = 'English'
+
+    if len(attributes['format']) == 0:
+        del attributes['format']
+    return attributes
+
+
+def get_showings_date(cinema_id: str, cinema: Cinema) -> [Showing]:
+    url = SHOWINGS_API_URL.format(cinema_id=cinema_id)
+    r = get_response(url)
+    try:
+        showings_data = r.json()
+    except json.JSONDecodeError:
+        raise CinemaArchiverException(f'Error decoding JSON data from URL {url}')
+
+    showings = []
+    for film_data in showings_data:
+        film_name = film_data['Title']
+        film_release_date = film_data['ReleaseDate']
+        film_year = str(datetime.fromisoformat(film_release_date).year)
+        film = Film(film_name, film_year)
+        for showing_day in film_data['Sessions']:
+            date = showing_day['NewDate']
```

Comment:
Please skip films beyond a certain date period, to avoid fetching films too far into the future.

Reply:
As discussed, no longer doing this in archivers.